addons/web/views/webclient_templates.xml

```diff
@@ -352,19 +352,23 @@
                 <t t-call-assets="web.assets_common" t-css="false"/>
                 <t t-call-assets="web.assets_frontend" t-css="false"/>
             </t>
-            <t t-set="body_classname" t-value="'container'"/>
-
-            <div class="row">
-                <div class="col-lg-6 offset-lg-3 o_database_list">
-                    <div class="text-center">
-                        <img t-attf-src="/web/binary/company_logo{{ '?dbname='+db if db else '' }}" alt="Logo"/>
-                    </div>
-                    <t t-raw="0"/>
-                    <div class="text-center" t-if="not disable_footer">
-                        <t t-if="not disable_database_manager">
-                            <a class="" href="/web/database/manager">Manage Databases</a> |
-                        </t>
-                        <a href="https://www.odoo.com?utm_source=db&amp;utm_medium=auth" target="_blank">Powered by <span>Odoo</span></a>
+            <t t-set="body_classname" t-value="'bg-100'"/>
+
+            <div class="container mb-5 pb-5">
+                <div class="row justify-content-center no-gutters">
+                    <div class="col-sm-8 col-md-6 col-lg-4 o_database_list">
+                        <div t-attf-class="mt-3 mt-lg-5 pt-3 pt-lg-4 px-2 px-sm-4 pb-3 {{login_box_classes}}">
```

Comment:
I did not see the inner div here... I do not understand the need to remove automatic gutters to add them back there :p But whatever.

Reply:
I'm not adding back the gutters :/
I'm defining a responsive padding so that the box will have always the same size regardless if it's transparent (community) or `.card` (enterprise)

Reply:
Mmh. In fact... you are making an exception here too :smile: You are using a card without a card-body... because you want different paddings for the card body. I like your design, it just seems there are more exceptions than standard content :smile: (Also, there is no point in having the same padding for community and enterprise :wink:)

I suggest this:
- Always use a card and a card-body, using standard BS structure (using a card in community would have the same look anyway if you remove the border, right?). If you are not happy with the card-body padding and you think you have to make an exception for that one only, I will let you do it.
- Set the size of the card using inline style as BS4 suggests: https://getbootstrap.com/docs/4.1/components/card/#using-custom-css (but use a max-width instead of a width to be responsive) (that way it will always have the size you want, no matter the screen)
- Do not use the grid system OR use col-auto (as you set the card width, the col will be the size of the card and the no-gutters class can be removed too)

What do you think?

Reply:
Cool, I'll try this approach, thanks.
Just a note
> there is no point in having the same padding for community and enterprise

I did that especially to avoid that the input fields touch the device's border in xs (in both versions)... but if it's not clear looking at the code  it's probably better to rethink this layout as you suggested ;)